Revit_Engine/Query/Names.cs

```diff
@@ -34,6 +35,11 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
+        [Description("Returns a collection of parameter names associated with a given type property inside MapSettings.")]
+        [Input("mapSettings", "MapSettings to be queried.")]
+        [Input("type", "Type to be sought for.")]
+        [Input("name", "Property name to be sought for.")]
+        [Output("names")]
         public static IEnumerable<string> Names(this MapSettings mapSettings, Type type, string name)
```

Comment:
Is the input `name` what the output `names` is? Don't think I get this one

Reply:
You look for Revit parameter names that are linked with a certain type property name, I agree that this does not look good, but it is legacy to be revisited in the coming Milestone - I would be comfortable with leaving it as is for now and giving it a deeper thought in a month from now.